frontend/src/components/Sidepanel/Sidepanel.tsx

```diff
@@ -48,22 +47,18 @@ const panels: PanelItem[] = [
 const Sidepanel = () => {
   const [activePanel, setActivePanel] = useState<PanelItem>()
 
-  const projectType = useProjectStore(s => s.project.config.type)
-
   // Open panel via event
   useEvent('sidepanel:open', e => {
     const panel = panels.find(p => p.value === e.detail.panel)
     setActivePanel(activePanel?.value === panel.value ? undefined : panel)
   }, [activePanel])
 
-  // Show bottom panel with TM Tape Lab
+  // Show bottom panel with TM Tape Lab (don't need to check if its TM, Editor.tsx:99 handles this)
```

Comment:
I more meant remove the check so we can use the bottom panel with other stuff in the future lol